src/sourceMaps/sourceMapUrl.ts

```diff
@@ -0,0 +1,57 @@
+/*---------------------------------------------------------
+ * Copyright (C) Microsoft Corporation. All rights reserved.
+ *--------------------------------------------------------*/
+
+ import { PossiblyCustomerContent, CustomerContent, NonCustomerContent } from '../chrome/logging/gdpr';
+import { logger } from 'vscode-debugadapter';
+import { SourceMapContents } from './sourceMapContents';
+
+export class SourceMapUrl {
+    private readonly _url: PossiblyCustomerContent<string>;
+    public readonly isInlineSourceMap: boolean;
+
+    public constructor(url: string) {
+        this.isInlineSourceMap = url.indexOf('data:application/json') >= 0;
```

Comment:
Is this the only time we'd consider the url customer content? I can think of other times where potentially PII could be passed in, for example, url params.

Reply:
You are right, those can also be potentially customer content. For the purpose of this PR we are focusing on source code only.

Reply:
got it